frontend/src/metabase/public/containers/PublicDashboard.jsx

```diff
@@ -127,6 +126,30 @@ class PublicDashboard extends Component {
     }
   }
 
+  getCurrentTabDashcards = () => {
+    const { dashboard, selectedTabId } = this.props;
+    if (!Array.isArray(dashboard?.dashcards)) {
+      return [];
+    }
+    if (!selectedTabId) {
+      return dashboard.dashcards;
+    }
+    return dashboard.dashcards.filter(
+      dc => dc.dashboard_tab_id === selectedTabId,
```

Comment:
nit: I like explicit names more than implicit names. Even though they would definitely be longer, I don't have to think twice what they actually represent.

I see there are a few of abbreviated names here
- dc
- pm
- p

Reply:
Fair, fixed 👍